commands_for_exercises.txt

```diff
@@ -0,0 +1,63 @@
+##Logging to Discovery:
+ssh -Y <yourusername>@login.discovery.neu.edu
+
+##Copying training material to Discovery using 'scp':
+git clone <url>
+
+##Exercise 1:
+# During training:
+srun --partition=reservation --reservation=bootcamp_gpu_2022 --nodes=1 --cpus-per-task=4 --gres=gpu:1 --mem=10G --time=02:00:00 --pty /bin/bash
+
+# Outside of training:
+srun --partition=gpu --nodes=1 --cpus-per-task=4 --gres=gpu:1 --mem=10G --time=02:00:00 --pty /bin/bash
+
+# Steps for building environment in your $HOME directory:
+module list
+module load cuda/11.0
+module load anaconda3/2021.05
+conda create --name pytorch_env python=3.7 anaconda -y 
+
+## Exercise 1:
+# If not already loaded:
+module load cuda/11.0
+module load anaconda3/2021.05
+
+# Load the environment:
+source activate pytorch_env
+# or
+source activate /shared/centos7/anaconda3/2021.05/envs/pytorch_env_training
+
+# Install gpu-enabled PyTorch inside the virtual environment:
+conda install pytorch torchvision torchaudio cudatoolkit=11.0 -c pytorch -y
+
+# test if GPU device is detected with PyTorch:
+python -c'import torch; print(torch.cuda.is_available())’
```

Comment:
python -c 'import torch; print(torch.cuda.is_available())' 

The (') at the end of the statement is the wrong one or formatted differently and it causes it to open python interpreter.

Reply:
This also has been updated @joseph-cooke